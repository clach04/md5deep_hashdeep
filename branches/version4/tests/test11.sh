#!/bin/sh
/bin/rm -rf /tmp/test11 /tmp/test11.out
mkdir /tmp/test11
cp ../testfiles/* /tmp/test11
hashdeep -r /tmp/test11 > /tmp/test11.out
if ! diff test11.out /tmp/test11.out ; 
then 
  echo test11 failed.
  exit 1
else
  exit 0
fi
