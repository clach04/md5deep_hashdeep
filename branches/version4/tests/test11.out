82ad3a128c5a4395a4d41c947b3a5499  /tmp/test/copying.txt offset 0-511
7272613466d441c436c7a141042f5445  /tmp/test/copying.txt offset 512-1023
8d9bf677118b91929cac777686a1e3f1  /tmp/test/copying.txt offset 1024-1535
6dca9fa816ba2c27cbd9c3d57cccf0b2  /tmp/test/copying.txt offset 1536-2047
ccfd5e0d4550bb24f88eb7b3a5766ed3  /tmp/test/copying.txt offset 2048-2559
1e32cc533f0a16c386af18ef1e86e0ff  /tmp/test/copying.txt offset 2560-3071
4266f321be5eb8f08c9a93df1398d1ab  /tmp/test/copying.txt offset 3072-3583
7a75807be820ef4a808ae7ccc339209b  /tmp/test/copying.txt offset 3584-4095
b7518f3a8f07b5ee0faef13b75f4b3ab  /tmp/test/copying.txt offset 4096-4607
8c3053f6a06c11be6f1cf107c752eb82  /tmp/test/copying.txt offset 4608-5119
52df9cdf09c5c5316ecc01911f5002ff  /tmp/test/copying.txt offset 5120-5631
a917e323250adaaa9f9b8d64e3634a15  /tmp/test/copying.txt offset 5632-6143
a7c29d80f30ede806353149ccd0b577a  /tmp/test/copying.txt offset 6144-6655
a4bda444a1b0543d6068259cf61acf3a  /tmp/test/copying.txt offset 6656-7167
0ab3bf1b6020ea21c4c348cc1eb45966  /tmp/test/copying.txt offset 7168-7679
86416d2fe5bb8eb75c549609490251c3  /tmp/test/copying.txt offset 7680-8191
f12899a1ec1df4bee057481d95a40eb8  /tmp/test/copying.txt offset 8192-8703
385fd0b1b32dc44e1b7ba92e2a0c549d  /tmp/test/copying.txt offset 8704-9215
ad9197e81ee99f19283f36902a29eeff  /tmp/test/copying.txt offset 9216-9727
68c7079d6d864fb9e2980dacd022c017  /tmp/test/copying.txt offset 9728-10239
aa09137042f825fb7613424aa300065f  /tmp/test/copying.txt offset 10240-10751
b71f0d94ddd12b68136823e65b82f05d  /tmp/test/copying.txt offset 10752-11263
9ff787551b3a906848403353b8ad5400  /tmp/test/copying.txt offset 11264-11775
148fe36b460feccc129cc5801fa7a960  /tmp/test/copying.txt offset 11776-12287
1f2ac4fed812dd5f0b524b50333ae98f  /tmp/test/copying.txt offset 12288-12799
303fd6c8c9851e62abf12668477cb4a1  /tmp/test/copying.txt offset 12800-13311
e891f7bf8b1e88daaf12644a64a939f0  /tmp/test/copying.txt offset 13312-13823
f79a22fe25a751e4e3270dac41bdd0fc  /tmp/test/copying.txt offset 13824-14335
e27cf76aa6058da1998dc6e19bcbd3f0  /tmp/test/copying.txt offset 14336-14847
ac9cbe61d970250e907c3dcf24d4f5c9  /tmp/test/copying.txt offset 14848-15359
95931146a77aef3e5df233a577e96149  /tmp/test/copying.txt offset 15360-15871
c7dd266fc6b0ba6132df105b893269d3  /tmp/test/copying.txt offset 15872-16383
379643ba852a62b5a4c13ea9ccb32543  /tmp/test/copying.txt offset 16384-16895
cbcbf912454f93e708e6ee733cd63398  /tmp/test/copying.txt offset 16896-17407
1c174b99b9849e92b0e80531bf797547  /tmp/test/copying.txt offset 17408-17919
4aa207bcf6a243dd30b995b9abe14444  /tmp/test/copying.txt offset 17920-18431
89f619e74f770419a81898c02dbb7e16  /tmp/test/copying.txt offset 18432-18943
656e7f44ce9d0bf3a9c39826bc5ee0c6  /tmp/test/copying.txt offset 18944-19123
deadbeefb303ba89ae055ad0234eb7e8  /tmp/test/deadbeef.txt offset 0-11
d472f529689b334bc4896828854e84ed  /tmp/test/foo.txt offset 0-43
da4b0deb0a440bce860bc0b6a239e26f  /tmp/test/install.txt offset 0-511
aa436dfc16536114e3a42379b423b829  /tmp/test/install.txt offset 512-1023
7af4435dce89e839b0c47085697fb942  /tmp/test/install.txt offset 1024-1535
4044d346c15c9cacddc1def7ea7087a5  /tmp/test/install.txt offset 1536-2047
15ca3e2ff3aa88b4289c5e86ff1c0438  /tmp/test/install.txt offset 2048-2559
960e6eb784c227ae30c6f929a2073755  /tmp/test/install.txt offset 2560-3071
be2a912e2ba97028999c86d9da6fd4a1  /tmp/test/install.txt offset 3072-3583
7411b52b502cc27152fc6d3ab645b065  /tmp/test/install.txt offset 3584-4095
37fdf7c178e50f899b61fa7603e446ad  /tmp/test/install.txt offset 4096-4607
058cc1871c4e4f75f2ca57ea25899e3c  /tmp/test/install.txt offset 4608-5119
522cd8d898ebefc1979af6f5c6cfe2f7  /tmp/test/install.txt offset 5120-5631
aa7884c3ccd376610d2b098243c84de2  /tmp/test/install.txt offset 5632-6143
356acb28ae70f041d24f88d63c4d1ee4  /tmp/test/install.txt offset 6144-6655
70864e0b431dc7e2f5f16729d083616c  /tmp/test/install.txt offset 6656-7167
d9c4e10f59384363db43603a78d75b42  /tmp/test/install.txt offset 7168-7679
55a2ff8185b2d31e39b1d2eb8e6252cf  /tmp/test/install.txt offset 7680-8191
90863a02be74f05dca1b1081f1e7cc1e  /tmp/test/install.txt offset 8192-8703
a745bf547ed9613dc112ba1008d6d798  /tmp/test/install.txt offset 8704-9215
26254ff5562de6f95e8bc43a792594e1  /tmp/test/install.txt offset 9216-9235
c4ca4238a0b923820dcc509a6f75849b  /tmp/test/one-byte-file offset 0-0
d41d8cd98f00b204e9800998ecf8427e  /tmp/test/zero-byte-file offset 0-18446744073709551615
